src/commands/config.js

```diff
@@ -1,35 +1,33 @@
-import { update as updateApplication } from '@clevercloud/client/esm/api/v2/application.js';
-
 import * as Application from '../models/application.js';
 import * as ApplicationConfiguration from '../models/application_configuration.js';
+import { Logger } from '../logger.js';
+import colors from 'colors/safe.js';
 
-import { sendToApi } from '../models/send-to-api.js';
+export async function list (params) {
+  const { alias, app: appIdOrName } = params.options;
+  const { ownerId, appId } = await Application.resolveId(appIdOrName, alias);
+  const app = await Application.get(ownerId, appId);
+  ApplicationConfiguration.printAllValues(app);
+}
 
 export async function get (params) {
   const [configurationName] = params.args;
   const { alias, app: appIdOrName } = params.options;
   const { ownerId, appId } = await Application.resolveId(appIdOrName, alias);
   const app = await Application.get(ownerId, appId);
-
-  if (configurationName == null) {
-    ApplicationConfiguration.print(app);
-  }
-  else {
-    ApplicationConfiguration.printById(app, configurationName);
-  }
+  ApplicationConfiguration.printValue(app, configurationName);
 }
 
 export async function set (params) {
   const [configurationName, configurationValue] = params.args;
   const { alias, app: appIdOrName } = params.options;
   const { ownerId, appId } = await Application.resolveId(appIdOrName, alias);
   const config = ApplicationConfiguration.getById(configurationName);
-
-  if (config != null) {
-    const app = await updateApplication({ id: ownerId, appId }, { [config.name]: ApplicationConfiguration.parse(config, configurationValue) }).then(sendToApi);
-
-    ApplicationConfiguration.printById(app, configurationName);
-  }
+  const patchedApp = {
+    [config.name]: ApplicationConfiguration.parse(config, configurationValue),
+  };
+  const app = await Application.updateOptions(ownerId, appId, patchedApp);
+  Logger.printSuccess(`Config ${colors.grey(config.id)} successfully updated to ${colors.grey(ApplicationConfiguration.formatValue(config, app[config.name]))}!`);
```

Comment:
On success message I previously used same color as the icon for text to highlight

```suggestion
  Logger.printSuccess(`Config ${colors.green(config.id)} successfully updated to ${colors.green(ApplicationConfiguration.formatValue(config, app[config.name]))}!`);
```